### ISSUE TITLE
Text not included in an ARIA landmark - SiteImprove - Drupal Announce

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa801#9045]
SI flagging an ARIA issue.

https://my2.siteimprove.com/Inspector/995867/A11Y/Page?pageId=25148795285&impmd=264ED37B65FA6186129334A508391970&issue=cantTell+failed&wcag=twopointone#/sia-r57/failed/Ylppb2VlbWg=;YTEwYzRhMjk=

Ensure all perceivable text is included in an ARIA landmark.

**Comment by joewhitsitt** [comment]
I believe this is related to the Drupal.announce div and not something we directly control. Unlike most sites, there isn't an announcement for most pages on load (siteimprove's scan window) which is why this is only showing for registrar on this page ("Fetching events").
